packages/playground-examples/copy/ja/3-8/Syntax and Messaging/Private Class Fields.ts

```diff
@@ -0,0 +1,67 @@
+//// { compiler: { ts: "3.8.3" } }
+// 3.8ではprivateフィールドが追加されました。これはサブクラスを含め、包括しているクラスの外側からは
+// アクセスできないようにするクラスフィールドの宣言方法です。
```

Comment:
包括としているというよりそのフィールドを包んでいるという意味かと思い、そっちで訳してみました
```suggestion
// 3.8ではprivateフィールドが追加されました。これはサブクラスを含め、フィールドを定義しているクラスの外側からは
// アクセスできないようにするクラスフィールドの宣言方法です。
```